File: src/main/java/um/haberes/core/model/internal/OrdenPagoRequest.java
package um.haberes.core.model.internal;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import java.math.BigDecimal;
import java.time.OffsetDateTime;

@Getter
@Setter
@EqualsAndHashCode
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class OrdenPagoRequest {

    private Integer anho = null;

    private Integer mes = null;

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssXX", timezone = "UTC")
    private OffsetDateTime fechaPago = null;

    private BigDecimal totalSantander = null;

    private BigDecimal totalOtrosBancos = null;

    private boolean individual = false;
}

File: src/main/java/um/haberes/rest/kotlin/model/internal/OrdenPagoRequest.kt
package um.haberes.rest.kotlin.model.internal

import com.fasterxml.jackson.annotation.JsonFormat
import java.math.BigDecimal
import java.time.OffsetDateTime

data class OrdenPagoRequest(

    var anho: Int? = null,
    var mes: Int? = null,
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ", timezone = "UTC")
    var fechaPago: OffsetDateTime? = null,
    var totalSantander: BigDecimal? = null,
    var totalOtrosBancos: BigDecimal? = null,
    var individual: Boolean = false

)
